### PR TITLE
proof-tools: Bump op-challenger version

### PR DESCRIPTION
**Pull request by Inphi**
Depends on https://github.com/ethereum-optimism/optimism/pull/12016 and https://github.com/ethereum-optimism/optimism/pull/12018

### CODE REVIEW

File: docker-bake.hcl
Comment (ajsutton) [inline docker-bake.hcl]: For the record, we don't always need this to be an actual challenger release.  It's fine to set it to one of the scheduled publishes from develop to pull in interim changes and then tag the proofs-tools image as a beta.  We can move everything to stable versions before tagging a stable proofs-tools release when we get to the point where that's actually needed. Right now proofs-tools is only used for testing.

But also, this version doesn't exist.  There's a release tag for v1.1.1 pending approval in circleci but I don't see a v1.1.2 at all.

Comment (Inphi) [reply docker-bake.hcl]: v1.1.2 is the optimistic ref that I was going to tag after #12016 merges. Noted. I'll replace this with a git hash.

Comment (ajsutton) [reply docker-bake.hcl]: Ah right.  The only catch with a git hash is that we don't publish every commit.  There's a `scheduled-docker-publish` job that runs every hour to publish docker images.

Comment (Inphi) [reply docker-bake.hcl]: Looks like that job can be manually triggered too. Not bad.

**Review by ajsutton — APPROVED** [review]
LGTM.

File: docker-bake.hcl
Comment (Inphi) [reply docker-bake.hcl]: Fixed. Using the latest scheduled-docker-publish tags - https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/65349/workflows/a29e3fb6-bbc1-47ca-bf69-b3997981da1b/jobs/2708576/steps

**Pull request merged**
